rules/prefer-object-define-properties.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+const isSameReference = require('./utils/is-same-reference.js');
+const {removeArgument} = require('./fix/index.js');
+
+const MESSAGE_ID = 'prefer-object-define-properties';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` over multiple `{{value}}`.',
+};
+
+function getFirstExpression(node, sourceCode) {
```

Comment:
This function only works for `ExpressionStatement`, you'll get unexpected result without checking node type

```js
foo(
	Object.defineProperty({}, "a", {}),
	Object.defineProperty({}, "b", {}),
);
```